label_studio/tasks/functions.py

```diff
@@ -31,15 +31,16 @@ def calculate_stats_all_orgs(from_scratch, redis):
             redis_job_for_calculation, org, from_scratch,
             redis=redis,
             queue_name='critical',
-            job_timeout=3600 * 24  # 24 hours for one organization
+            job_timeout=3600 * 24,  # 24 hours for one organization
+            migration_name=migration_name
         )
 
         logger.debug(f"Organization {org.id} stats were recalculated")
 
     logger.debug("All organizations were recalculated")
 
 
-def redis_job_for_calculation(org, from_scratch):
+def redis_job_for_calculation(org, from_scratch, migration_name=''):
```

Comment:
default name should be 0018_manual_migrate_counters

Reply:
Fixed